.github/workflows/verify-ec2.yml

```
@@ -0,0 +1,71 @@
+---
+name: CIS AWS Foundations v2.0.0
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  my-job:
+    name: Validate my profile
+    runs-on: ubuntu-latest
+    env:
+      CHEF_LICENSE: accept-silent
+      CHEF_LICENSE_KEY: ${{ secrets.SAF_CHEF_LICENSE_KEY }}
+      RESULTS_FILE: ${{ github.sha }}-inspec_results.json
+      PROFILE_FILE: profile.json
+      INPUT_FILE: testing.inputs.yml
+      HEIMDALL_URL: https://heimdall-demo.mitre.org
+    steps:
+      - name: add needed packages
+        run: sudo apt-get install -y jq
+      - name: Configure AWS credentials
+        env:
+          AWS_SG_ID: ${{ secrets.SAF_AWS_SG_ID }}
+          AWS_SUBNET_ID: ${{ secrets.SAF_AWS_SUBNET_ID }}
+
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.SAF_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.SAF_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Clone full repository so we can push
+        run: git fetch --prune --unshallow
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.1"
+      - name: Disable ri and rdoc
+        run: 'echo "gem: --no-document" >> ~/.gemrc'
+      - run: bundle install
+      - name: Installed Inspec
+        run: bundle exec inspec version
+      - name: Vendor the InSpec Profile
+        run: bundle exec inspec vendor --overwrite
+      - name: Lint the Inspec profile
+        run: bundle exec inspec check .
+      - name: Run the Profile
+        run: |
+          bundle exec inspec exec . --target aws:// --input-file=${{ env.INPUT_FILE }} --reporter cli json:${{ env.RESULTS_FILE }} --enhanced-outcomes --filter-empty-profiles || true
+      - name: Save Test Result JSON
+        uses: actions/upload-artifact@v3
+        with:
+          path: |
+            ./${{ env.RESULTS_FILE }}
+      - name: Display our ${{ env.RESULTS_FILE }} results summary
+        uses: mitre/saf_action@v1
+        with:
+          command_string: "view summary -i ${{ env.RESULTS_FILE }}"
+      - name: Ensure ${{ env.RESULTS_FILE }} meets our results threshold
+        uses: mitre/saf_action@v1
+        with:
+          command_string: "validate threshold -i ${{ env.RESULTS_FILE }} -F cis-foundations.threshold.yml"
+      
+      - name: Upload to Heimdall
```

Review this or simular to see if it may be useful: https://stackoverflow.com/questions/69774778/how-to-use-curls-response-in-other-run-commands-in-github-actions
```suggestion
      - name: Upload to Heimdall
```